### PR TITLE
add support css attribute, bug fix child selector

### PR DESCRIPTION
**Pull request by taiyonoryoshy**
add support css attribute (.a[href*="site.com"]). bug fix children elements.

### CODE REVIEW

**Comment by oliverklee** [comment]
Hi @taiyonoryoshy, thanks a lot for the PR! :heart: 

However, I'd prefer to have one bug fix/feature per PR. Could you please split up you PR into several separate PRs and also add unit tests that fail without your change and pass with the change?

You might also want to look the previous commit message to get a feeling for the format of the commit messages which we prefer.

Again, thanks for contributing!

File: Classes/Emogrifier.php [hunk 1]
```diff
@@ -159,21 +159,31 @@ class Emogrifier
      */
     private $xPathRules = [
         // child
-        '/\\s+>\\s+/'                              => '/',
+      '/\\s*>\\s*/'                                                                  => '/',
```
Comment (oliverklee) [inline Classes/Emogrifier.php]: Please keep the indentation to 4 spaces (so the code will be PSR-2-compliant).

**Comment by oliverklee** [comment]
Closing this PR as this PR mixes up the feature and the bug fixes. @taiyonoryoshy, could you please open separate PRs for the separate things? Thanks!

**Comment by taiyonoryoshy** [comment]
@oliverklee I will add unit tests and open separate PRs for the separate things. I will write later.

**Comment by oliverklee** [comment]
Thanks!

**Pull request closed without merging**